service/src/main/java/greencity/mapping/AddServiceDtoMapper.java

```diff
@@ -0,0 +1,31 @@
+package greencity.mapping;
+
+import greencity.dto.AddServiceDto;
+import greencity.dto.TariffTranslationDto;
+import greencity.entity.order.Bag;
+import greencity.entity.order.BagTranslation;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AddServiceDtoMapper extends AbstractConverter<Bag, AddServiceDto> {
+    @Override
+    protected AddServiceDto convert(Bag source) {
+        List<BagTranslation> translations = new ArrayList<>(source.getBagTranslations());
+        List<TariffTranslationDto> dto = translations.stream().map(
+            i -> new TariffTranslationDto(i.getName(), i.getDescription(), i.getLanguage().getId()))
+            .collect(Collectors.toList());
+        source.setFullPrice(source.getPrice() + source.getCommission());
```

Comment:
Why are you modifying state of the source in Mapper?
If you need to have full price calculated - define method `getFullPrice` in class `Bag` that will be adding price to commision.
Another problem I see here - you calculate full price but do not include it in `AddServiceDto`